app/components/Article/index.tsx

```diff
@@ -158,15 +162,9 @@ const ArticleFooter = (question: Question) => {
       </div>
       <span>Did this page help you?</span>
 
-      <CompositeButton>
-        <Button className="secondary" action={() => alert('Like')}>
-          <ThumbUpIcon />
-          <span className="teal-500">Yes</span>
-        </Button>
-        <Button className="secondary" action={() => alert('Dislike')}>
-          <ThumbDownIcon />
-          <span className="teal-500">No</span>
-        </Button>
+      <CompositeButton style={{display: 'flex'}}>
```

Comment:
don't do inline styles. There is a class for this, so just do `className="flex"`